--- lastbash-0.3.0/Makefile~	2006-12-06 20:39:45.000000000 +0100
+++ lastbash-0.3.0/Makefile	2006-12-06 20:39:42.000000000 +0100
@@ -11,9 +11,9 @@
 doclist = AUTHORS COPYING README INSTALL TODO VERSION ChangeLog
 
 # Directories
-prefix = /usr/local
+prefix = /usr
 bindir = $(DESTDIR)$(prefix)/bin
-mandir = $(DESTDIR)$(prefix)/man
+mandir = $(DESTDIR)$(prefix)/share/man
 man1dir = $(mandir)/man1
 datadir = $(DESTDIR)$(prefix)/share/$(name)
 srcdir = .
